lithium/lithium.py

```diff
@@ -929,23 +928,22 @@ def tryArgumentsAsGlobals(self, roundNum, testcase, interesting, tempFilename):
             for match in re.finditer(br'((\w+)\s*\(((?:[^()]|\([^,()]*\))*)\))', line):
                 pattern = match.group(1)
                 fun = match.group(2)
-                if match.group(3) == "":
+                if match.group(3) == b"":
                     args = []
                 else:
-                    args = match.group(3).split(',')
+                    args = match.group(3).split(b',')
                 if not fun in functions:
                     functions[fun] = { "uses": [] }
                 functions[fun]["uses"] += [{ "values": args, "chunk": chunk, "pattern": pattern }]
 
-
         # All patterns have been removed sucessfully.
         if len(functions) == 0 and len(anonymousQueue) == 0:
             return 0, testcase
 
         log.info("Starting removing function arguments.")
 
         for fun, argsMap in functions.items():
-            description = "arguments of '" + fun + "'"
+            description = "arguments of '%s'" % fun.decode("utf-8", "replace")
```

Comment:
Do we have other parts of the code which requires this `.decode("utf-8", "replace")` section?